images/dhcp/Containerfile

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/openshifttest/base-alpine:1.2.0
 
-RUN apk add --no-cache dnsmasq
+RUN rpm-ostree install dnsmasq && \
```

Comment:
I see now, you wanted to build dnsmasq on top of base-alpine, but then you replaced the from statement in the testconfigs.

Reply:
Can you change this to build on top of cs9?

Reply:
Also adding some comments on the other pr.

Reply:
done